src/config/rounds/round1.spec.tsx

```diff
@@ -58,55 +52,44 @@ describe('round 1', () => {
         const game = getGame();
 
         game.selectAction('ACTION_PROTECTED_FROM_OUTSIDE_DISTRACTION');
+        expect(game.state.currentRound.userStoryChance).toEqual(40);
 
-        // TODO Hannes - very very confused as to how/what this is checking
-        // it would help if this was extracted to a testHelper so it could be used to test multiple actions
-        addRolls(Array(game.state.currentRound.capacity.available).fill(0.39));
-        game.nextRound();
-        expect(game.state.result.storiesCompleted).toEqual(9);
-
-        expect(game.state.currentRound.capacity.available).toEqual(10);
-
-        addRolls(Array(5).fill(0.39));
-        addRolls(Array(5 + /* gremlin */ 2).fill(0.41));
-        game.nextRound();
-        expect(game.state.result.storiesCompleted).toEqual(14);
```

Comment:
The above was implicitly testing the implementation of `storySucceeds` which was bad because it required implementation knowledge in this test. moved that out to `src/lib/storySucceeds.spec.ts` 

I also made `userStoryChance` available on `state.currentRound` so that the current chance can be tested explicitly.